service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -84,6 +84,9 @@ class CaseData implements JsonSerializable
     #[Annotation\Required(false)]
     public ?string $searchPostcode = null;
 
+    #[Annotation\Required(false)]
+    public ?array $id_method_intl = [];
```

Comment:
It's not really clear to me what this new property is for, could it be renamed to have a clearer distinction from `idMethod`?

Should also be camelCase to match the other properties.